pkg/sql/crdb_internal.go

```diff
@@ -2452,12 +2452,12 @@ CREATE TABLE crdb_internal.zones (
 				}
 
 				for i, s := range subzones {
-					index, err := table.FindIndexByID(sqlbase.IndexID(s.IndexID))
-					if err != nil {
-						if errors.Is(err, sqlbase.ErrIndexGCMutationsList) {
-							continue
-						}
-						return err
+					index := table.FindActiveIndexByID(sqlbase.IndexID(s.IndexID))
```

Comment:
if you were fussed with performance, we should cache index ids by table descriptor ids, and do a lookup based on that. this seems index^2-y.